metron-deployment/vagrant/hackathon-image-vagrant/VagrantFile

```diff
@@ -0,0 +1,94 @@
+#
```

Comment:
@merrimanr  - could you change the filename to Vagrantfile (small 'f'). It will mostly work, but evidently there are some corner cases where the filename is strictly case sensitive and needs to be Vagrantfile.

[https://www.vagrantup.com/docs/vagrantfile/](url)

Thanks!